.circleci/config.yml

```diff
@@ -1,11 +1,11 @@
 version: 2
 jobs:
   build:
-    working_directory: /go/src/github.com/jszwedko/go-circleci
+    working_directory: /go/src/github.com/mattermost/go-circleci
     environment:
       TEST_RESULTS: /tmp/test-results
     docker:
-    - image: circleci/golang:1.10
+    - image: circleci/golang:1.13
```

Comment:
Any reason we can't go for 1.14?

Reply:
Just to be conservative by now and use the minimum required version

Reply:
The minimum required version is actually 1.14 for server. Which minimum required version are you referring to?

Reply:
To the use of `RequestWithContext` given that this project could be used by other projects that are not tied to 1.14 but I can change to that version without problem

Reply:
Ah no worries. This is fine for now. Thanks.